test_files/enum.ts

```diff
@@ -1 +1,3 @@
-enum FooEnum {XYZ, PI = 3.14159}
\ No newline at end of file
+enum EnumTest1 {XYZ, PI = 3.14159}
+
+export enum EnumTest2 {XYZ, PI = 3.14159}
```

Comment:
Actually, maybe add a comment to explain why an exported enum needs separate testing.